lib/tasks/add_transcripts_to_oral_histories.rake

```diff
@@ -0,0 +1,57 @@
+namespace :scihist do
+  desc """
+    Goes through all the oral histories and adds transcripts from a file on the disk, for those missing them:
+
+    bundle exec rake scihist:add_transcripts_to_oral_histories
+  """
+
+  task :add_transcripts_to_oral_histories => :environment do
+    files_location = '/tmp/ohms_transcript_files/'
+    progress_bar = ProgressBar.create(total: Work.where("json_attributes -> 'genre' ?  'Oral histories'").count, format: "%a %t: |%B| %R/s %c/%u %p%% %e")
+    Work.where("json_attributes -> 'genre' ?  'Oral histories'").find_each(batch_size: 10) do |w|
```

Comment:
Nice job figuring out how to use find_each here, to accomodate cases where we have very large numbers -- it's just the right way to do "iterate through everything". 

I don't think you need the `batch_size` argument, you can just leave it the Rails default. (Which yes, means that for our actual number of records, it won't be doing much batching at all -- that's fine, Rails knows what it can handle, generally, unless we have reason to know/believe otherwise).

Reply:
Removed!